docker-compose.yml

```diff
@@ -5,6 +5,10 @@ services:
     environment:
       POSTGRES_PASSWORD: postgres
       POSTGRES_HOST_AUTH_METHOD: md5
+    ports:
+      - "54321:5432"
```

Comment:
Plz use port number lower than 49152 as this can generate errors, cause dynamic port range may be used for outgoing connections. Gonna make a hard time for a fellow developer to debug why this docker-compose does not work